tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature

```diff
@@ -60,27 +59,29 @@ Feature: checking file versions using file id
   Scenario Outline: check the versions of a file after moving to a shared folder inside a project space as editor/viewer
     Given user "Alice" has created a folder "testFolder" in space "Project1"
     And user "Alice" has created a share inside of space "Project1" with settings:
-      | path      | testFolder   |
-      | shareWith | Brian        |
-      | role      | <space-role> |
+      | path      | testFolder |
+      | shareWith | Brian      |
+      | role      | <role>     |
     And user "Alice" has moved file "text.txt" to "/testFolder/movedText.txt" in space "Project1"
     When user "Alice" gets the number of versions of file "/testFolder/movedText.txt" using file-id path "/meta/<<FILEID>>/v"
     Then the HTTP status code should be "207"
     And the number of versions should be "1"
     When user "Brian" tries to get the number of versions of file "/Shares/testFolder/movedText.txt" using file-id path "/meta/<<FILEID>>/v"
     Then the HTTP status code should be "403"
     Examples:
-      | space-role |
-      | editor     |
-      | viewer     |
+      | role   |
+      | editor |
+      | viewer |
 
   @issue-7738
-  Scenario: check the versions of a file after moving it to a shared folder of a project space shared with all permissions
+  Scenario: check the versions of a file after moving it to a shared folder inside a project space as manager
     Given user "Alice" has created a folder "testFolder" in space "Project1"
     And user "Alice" has created a share inside of space "Project1" with settings:
       | path      | testFolder |
       | shareWith | Brian      |
-      | role      | all        |
+    And user "Alice" has shared a space "Project1" with settings:
```

Comment:
do we need both steps here?

Reply:
Yes it needed. User should be member of the space(editor or manager) to see file version. 
If I delete this step -> test will be failed on 87 line (403 error)

Reply:
ahh, it didn't notice the sahre of testFolder